scripts/dotnet-cli-build/Publishing/AzurePublisher.cs

```diff
@@ -187,5 +187,23 @@ public string CalculateArchiveBlob(string archiveFile, string channel, string ve
         {
             return $"{channel}/Binaries/{version}/{Path.GetFileName(archiveFile)}";
         }
+
+        public void DownloadFiles(string blobVirtualDirectory, string fileExtension, string downloadPath)
```

Comment:
I think DownloadFiles is fine here. Think of Directory.GetFiles(...) it takes a wildcard as well.